pkg/metrics/otel.go

```diff
@@ -54,13 +54,18 @@ exporters:
   googlecloud/kubernetes:
     metric:
       prefix: "kubernetes.io/internal/addons/config_sync/"
-      skip_create_descriptor: false
+      skip_create_descriptor: true
+      instrumentation_library_labels: false
+      create_service_timeseries: true
+      service_resource_labels: false
```

Comment:
I would recommend adding he CS-specific [resource attributes](https://github.com/GoogleContainerTools/kpt-config-sync/blob/208f34fde1d5cc3b05872319cdef89ff43a1dcb3/manifests/templates/reconciler-manager-configmap.yaml#L283-L292) into the [metric.resource_filters](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/googlecloudexporter) field, and test the new metrics in Prometheus and Cloud Monitoring to make sure things are working as expected.

Reply:
I'm thinking maintaining a list of CS resource label might be similar to excluding the exporter's special labels?

Testing ...

Reply:
The resource_filter is not working as expected on my end at this point. I would like to split this work to a different PR - just not sure if we still have enough time. 

Also looking at the spec in their doc:  If provided, `resource attributes` matching any filter will be included in `metric labels`, seems like there is a conversion from resource label to a custom label